x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts

```diff
@@ -58,6 +62,33 @@ export const getLogger = (endpointAppContext: EndpointAppContext): Logger => {
   return endpointAppContext.logFactory.get('metadata');
 };
 
+const errorHandler = <E extends Error>(
```

Comment:
Generic error handler for all metadata routes. Currently only being used in the GET single endpoint metadata api